cranelift/codegen/Cargo.toml

```diff
@@ -40,7 +40,7 @@ wast = { version = "29.0.0", optional = true }
 cranelift-codegen-meta = { path = "meta", version = "0.68.0" }
 
 [features]
-default = ["std", "unwind"]
+default = ["std", "unwind", "x64"]
```

Comment:
Please revert this. I am pretty sure that cranelif-module doesn't disable default features and as such it would become impossible to use the old backend.

Reply:
Eek, this slipped in from local testing. Thanks for catching this.

Reply:
(I think that's why I was newly seeing a warning issue; CI doesn't otherwise check for warnings with non-default options.)